### PR TITLE
Update the Content loader README

### LINKED ISSUE #412: Update content-loader README.md
**Description**

Review and update the content of the `content-loader` [README.md](https://github.com/kyma-project/website/blob/05a863e204a11493a6335af1b9513ccd65786f16/tools/content-loader/README.md) as there are some language mistakes.

**Reasons**

Updated documentation.

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Updated the Content loader README

**Related issue(s)**
Resolves #412

### CODE REVIEW

File: tools/content-loader/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Content loader
+# Content Loader
 
 ## Overview
 
-This project prepares content for the Docs on the `kyma-project.io` website. It uses GitHub API to detect new releases in the `kyma` repository and documentation changes under [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch. It later prepares the content for documentation and roadmap sections and publishes it on the website.
+The Content loader prepares content for the **Docs** and **Roadmap** section on the `kyma-project.io` website. It uses GitHub API to detect new releases in the `kyma` repository and documentation changes under [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch. Once detected, it prepares the content for the **Docs** and **Roadmap** sections and publishes it on the website.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply tools/content-loader/README.md]: I would either add `the` before `content` at the beginning of the sentence or remove it from the last part of the sentence: `it prepares the content for the **Docs** and **Roadmap** sections`.

[the remark this answers is not in the document]
Comment (mmitoraj) [reply tools/content-loader/README.md]: I will remove the `the` ;)

File: tools/content-loader/README.md [hunk 2]
```diff
@@ -1,12 +1,19 @@
-# Content loader
+# Content Loader
 
 ## Overview
 
-This project prepares content for the Docs on the `kyma-project.io` website. It uses GitHub API to detect new releases in the `kyma` repository and documentation changes under [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch. It later prepares the content for documentation and roadmap sections and publishes it on the website.
+The Content Loader prepares content for the **Docs**, **Community**, and **Roadmap** sections on the `kyma-project.io` website. It uses GitHub API to detect:
+
+- new releases in the `kyma` repository merged to the `master` branch,
```
Comment (kazydek) [inline tools/content-loader/README.md]:
```suggestion
- New releases in the `kyma` repository
```

File: tools/content-loader/README.md [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+- documentation changes under [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch,
```
Comment (kazydek) [inline tools/content-loader/README.md]:
```suggestion
- Documentation changes under [`/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch of the `kyma` repository
```

File: tools/content-loader/README.md [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+- documentation changes in the `community` repository merged to the `master` branch,
```
Comment (kazydek) [inline tools/content-loader/README.md]:
```suggestion
- Documentation changes merged to the `master` branch of the `community` repository
```

File: tools/content-loader/README.md [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+- GuHub issues with the `epic` label.
```
Comment (kazydek) [inline tools/content-loader/README.md]:
```suggestion
- GitHub issues with the `epic` label
```

File: tools/content-loader/README.md [hunk 6]
```diff
@@ -1,12 +1,19 @@
-# Content loader
+# Content Loader
 
 ## Overview
 
-This project prepares content for the Docs on the `kyma-project.io` website. It uses GitHub API to detect new releases in the `kyma` repository and documentation changes under [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) merged to the `master` branch. It later prepares the content for documentation and roadmap sections and publishes it on the website.
+The Content Loader prepares content for the **Docs**, **Community**, and **Roadmap** sections on the `kyma-project.io` website. It uses GitHub API to detect:
+
+- New releases in the `kyma` repository,
```
Comment (kazydek) [inline tools/content-loader/README.md]:
```suggestion
- New releases in the `kyma` repository
```

**Pull request merged**